INSTALL.md

```diff
@@ -337,9 +337,17 @@ After that, build the executable JAR file using the following command:
 mvn package
 ~~~
 
-The JAR file `mmj2-2.5.2-SNAPSHOT-jar-with-dependencies.jar` will be
-created with in the directory `mmj2/target` and can be run using
+The JAR file `mmj2-${project.version}-jar-with-dependencies.jar` will be
+created with in the directory `mmj2/target` and can be run directly using
 
 ~~~
-java -jar target/mmj2-2.5.2-SNAPSHOT-jar-with-dependencies.jar <mmj2-options>
+java -jar target/mmj2-${project.version}-jar-with-dependencies.jar <mmj2-options>
```

Comment:
Is this `${project.version}` syntax expected to be expanded by the reader to whatever the latest is? If so, you should give an example and point to the place where they can find out what actual value to put in there.

Reply:
I have modified the commits to include an explanation of the `${project.version}` syntax and where to find the version number that `${project.version}` stands for.